openfga/v1/openfga_service.proto

```diff
@@ -671,6 +671,106 @@ service OpenFGAService {
       description: "Returns a paginated list of OpenFGA stores."
     };
   }
+
+  // StreamedListObjects is a streaming variation of the ListObjects API (see ListObjects for more info).
+  rpc StreamedListObjects(StreamedListObjectsRequest) returns (stream StreamedListObjectsResponse) {
+    option (google.api.http) = {
+      post: "/stores/{store_id}/streamedListObjects",
```

Comment:
```suggestion
      post: "/stores/{store_id}/streamed-list-objects",
```

(to follow the same pattern as the rest of the APIs)

Reply:
Wish we had caught this in our RFC process :). Good call out.